<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogHexEditor</class>
 <widget class="QDialog" name="DialogHexEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hex Editor</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>448</width>
     <height>384</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
    </font>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string notr="true"/>
   </property>
   <property name="overwriteMode">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>8</y>
     <width>128</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>32</y>
     <width>128</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>128</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>124</y>
     <width>128</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Set value</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>148</y>
     <width>128</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Search for value</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>196</y>
     <width>128</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Equal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not equal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Less than</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Less or equal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Greater than</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Greater or equal</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Signed</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>236</y>
     <width>128</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Search backwards</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
